src/main/com/fulcrologic/rad/type_support/date_time.cljc

```diff
@@ -332,10 +334,11 @@
   "Convert an HTML date input string to an inst at the given local time, adjusted to the correct *current-timezone*. Returns
    `now` if the string isn't a proper ISO string."
   [html-date local-time]
-  [(? string?) ::local-time => inst?]
-  (let [date      (or (ld/parse html-date) (ld/now))
-        date-time (ld/at-time date local-time)]
-    (local-datetime->inst date-time)))
+  [(? string?) ::local-time => (? inst?)]
+  (when-not (= "" html-date)
```

Comment:
What if html-date is nil, which you've changed the spec to allow?

Reply:
Breaking change from docstring as well. Need to update the docstring to be correct if we're chaning this.

Reply:
I haven't changed the spec for input, it was allowed to be nil all along. 

In use it's not nil because the value of an input is not nil, and we probably should change the spec so that html-date is not allowed to be nil.

Reply:
The docstring was incorrect all along:
```
Returns  `now` if the string isn't a proper ISO string.
```
Then parsing `""` should return `(now), right? Here's what I get on `develop` branch:
```
(com.fulcrologic.rad.type-support.date-time/html-date->inst
  ""
  (cljc.java-time.local-time/now))
=> #object[JsJodaException DateTimeParseException: Text ''could not be parsed at index 0: at index: 0]
```

So the docstring is not correct as it is.

Reply:
Right, so the docstring needs a fix since no one was ever relying on that behavior I'm totally on-board with this one being a trivial accept as a bug fix.